### PR TITLE
First version of README.md

### PR DESCRIPTION
**Pull request by mzitnik**

### CODE REVIEW

**Comment by mzitnik** [comment]
This is the first version of README file

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<div align="center">
+<p>
+<img src="https://github.com/ClickHouse/clickhouse-js/blob/a332672bfb70d54dfd27ae1f8f5169a6ffeea780/.static/logo.svg" width="200px" align="center">
```
Comment (BentsiLeviav) [inline README.md:3]: If we add CH's logo, could you please add Beam's as well?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* Continues testing with [ClickHouse Cloud](https://clickhouse.com/cloud)
+* Example of using ClickHouseIO with Apache Beam (Complete Pipeline)
+
+## Continues testing
+To make sure that latest Apache Beam version works with ClickHouse Cloud we have created a Nightly git workflow that runs the e2e beam pipeline every day.
+This will make sure that the k latest versions of Apache Beam work with ClickHouse Cloud.
+
+## Examples 
+
+A Full Maven example with pom.xml definition see: [here](https://github.com/ClickHouse/clickhouse-beam-connector/tree/main/example/mvn)
```
Comment (BentsiLeviav) [inline README.md]: If possible, please move the URL from the word here to the word example.

**Review by BentsiLeviav — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mzitnik) [reply README.md:3]: I think we need have a ClickHouse logo with Beam inside

File: README.md (hunk 2 above)
Comment (mzitnik) [reply README.md]: @BentsiLeviav 
I will change to `mvn`  since we are going to have `gradle` in the future

**Review by BentsiLeviav — APPROVED** [review]

**Pull request merged**